api/apps/api/src/modules/blm/values/project-blm-repository.ts

```diff
@@ -0,0 +1,33 @@
+import { Either } from 'fp-ts/Either';
+
+export const unknownError = Symbol(`unknown error`);
+export const projectNotFound = Symbol(`project not found`);
+export const alreadyCreated = Symbol(`project already has defaults`);
+
+export type CreateFailure = typeof unknownError | typeof alreadyCreated;
+export type SaveFailure = typeof unknownError | typeof projectNotFound;
+export type GetFailure = typeof unknownError | typeof projectNotFound;
+
+export interface ProjectBlm {
+  id: string;
+  range: [number, number];
+  values: number[];
+  defaults: number[];
+}
```

Comment:
minor, but I would find it useful to add some inline documentation at least for this core interface.

A reference to the design documents may be enough, but having at least some core traits of this interface documented for quick in-IDE reference may a way to be kinder to our future selves, e.g. defaults is set-once, values is calculated from range, range is user-supplied... -  all this can also be gleaned from the interfaces in the rest of this PR, of course.

Reply:
@hotzevzl 👍  https://github.com/Vizzuality/marxan-cloud/pull/626/commits/32a887dcbbf9d986b3ca9284a4c032f4bcfc0a0d